### DISCUSSION TITLE
Remove unused files under `.github/workflows`

### DISCUSSION THREAD

**Discussion by myl7** [ustctug/ustcthesis#329] [General]
CI of the project uses `tunathu/thuthesis-test-env` image, which is built according to `.github/workflows/Dockerfile` of tuna/thuthesis. Since the project does not include the Dockerfile, `texlive.profile` and `install-texlive.sh` in `.github/workflows`, which is used only in Docker building, is unused in the project. I would suggest to remove them to avoid confusing users.

**Comment by zepinglee** [comment]
Yes, you are right. The `install-texlive.sh` and `texlive.profile` files were originally used for installing a TeX distribution for every run of CI. It was then switched to the docker image because of some problems with CTAN and those files are no longer in valid use.

**Comment by zepinglee** [comment]
Fixed in 6e7abb1.
